scripts/tests/doppelganger_protection.sh

```diff
@@ -126,30 +152,20 @@ if [[ "$BEHAVIOR" == "success" ]]; then
     sleep $(( $SECONDS_PER_SLOT * 32 * 2 ))
     for val in 0x*; do
         [[ -e $val ]] || continue
-        curl -s localhost:9100/lighthouse/validator_inclusion/5/$val | jq | grep -q '"is_previous_epoch_target_attester": true'
-        IS_ATTESTER=$?
-        if [[ $IS_ATTESTER -eq 0 ]]; then
+        resp=$(curl -s $bn2_2_host_url/lighthouse/validator_inclusion/5/$val)
+        # TODO: error handling
```

Comment:
I realised this is silently failing, hence we don't see any errors on CI sometimes. I'll add some error handing code.